<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>948</width>
    <height>549</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Servidor FME</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
border: 4px solid  #0f0f0f;
background:  #536872;
}

</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout2">
     <item>
      <widget class="QTabWidget" name="categoryTabWidget">
       <property name="styleSheet">
        <string notr="true">background:  #285663;
</string>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Buscar Rotina&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit { 
                         border-radius: 8px;
							border: 4px solid  #100c08;
                         color:rgb(0, 0, 0); 
							image: url(:/plugins/FmeServer/search.png);
							image-position: right;
							background-color: rgb(255, 255, 255);
							font: 12pt &quot;Monospace&quot;;

 }
QLineEdit:focus { 
                         border:4px outset; 
                         border-radius: 8px; 
                         border-color: #0892d0; 
                         color:rgb(0, 0, 0); 
 }</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout1">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
